README.md

```diff
@@ -1,13 +1,14 @@
 # rb
 
-With 9 lines of Ruby replace most of the command line tools that you use to process text inside of the terminal.
+With 10 lines of Ruby replace most of the command line tools that you use to process text inside of the terminal.
```

Comment:
Using the original way of counting LoCs it's actually `11` lines of Ruby now, not `10`   :smile:

Reply:
But if we decide to not count empty lines, it's actually `9` lines of *actual* code!  :wink: